### PR TITLE
House: Added House Creation Page

### PR DESCRIPTION
**Pull request by LiorKGOW**
- The page presents a form to create a new House.
- When the user completes the form, a new House is created, the user is directed to his Mine Page of his new House

Still need to add tests 👍 

Screenshots:

Empty Form when a user enters the House Creation Page:

Empty Form when a user enters the house creation page

An example of a good form being submitted by the user:

Example for a good form being submitted by the user

Creation of House directs to Mine Page with the new created house id:

The user being directed to his new house's mine page

The user's new created house is now shown in the global page (which extracts from the data base all the public houses)

The user's new created house is now shown in the global page (which extracts from the data base all the public houses)

An example of a form of a NON-Public house being submitted:

*insert*

Creation of House directs to Mine Page with the new created house id:

*insert*

The User can now login in the Login Page with his new House ID:

*insert*

Error:

?

### CODE REVIEW

File: templates/house_create/house_create.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}{% load static %}{%block content%}
+
+house creation
+
+{% endblock content %}
```
Comment (ronyyosef) [inline templates/house_create/house_create.html:5]: Missing empty line

**Review by ronyyosef — CHANGES_REQUESTED** [review]

**Comment by adiabramovitch** [comment]
Why not also add the template itself?

**Comment by ShellyGolden** [comment]
What does this PR do?
@LiorKGOW

**Comment by LiorKGOW** [comment]
rebased and repushed after #38 was merged

File: templates/house_create/house_create.html (hunk 1 above)
Comment (LiorKGOW) [reply templates/house_create/house_create.html]: > Missing empty line

added an empty line at the end of the file 
thanks👍

**Comment by LiorKGOW** [comment]
> Why not also add the template itself?

This PR will be the view of House Creation.
Still in production

**Comment by adiabramovitch** [comment]
Does this PR on draft mode because the create-house form is missing?

**Comment by LiorKGOW** [comment]
> Does this PR on draft mode because the create-house form is missing?

Because I wanted to finish with the Login Page first

**Comment by ronyyosef** [comment]
update here when it's ready for review

**Comment by ronyyosef** [comment]
@LiorKGOW can we close this PR?

**Comment by LiorKGOW** [comment]
> @LiorKGOW can we close this PR?

I am still working on it
hopefully today it will be ready for review 👍

**Comment by LiorKGOW** [comment]
rebased and repushed after resolving conflicts of #81 & #71 were merged to main 👍

**Comment by ronyyosef** [comment]
I added the create house to the Django user auth PR: #99 
Please take a look there and review it

**Comment by LiorKGOW** [comment]
> I added the create house to the Django user auth PR: #99 Please take a look there and review it

Closing this PR 👍

**Comment by LiorKGOW** [comment]
> > I added the create house to the Django user auth PR: #99 Please take a look there and review it
> 
> This Code is used in Rony's PR #99
> Closing this PR 👍

**Pull request closed without merging**